src/isr.cpp

```diff
@@ -174,10 +174,8 @@ void isr() {
 		}
 	}
 	// glides
-	// in monophonic mode, we glide only while the old note is still held down.
-	bool skip_glide = !preset_data.is_polyphonic() && voice_state.n_held_keys() <= 1;
```

Comment:
@twisted-electrons this was causing the bug, because `voice_state.n_held_keys()` only counts the keys held on the *master* channel. That means, when playing legato notes on channels 2,3,4, it comes to the conclusion "no legato notes on the main channel -> no glide"